bin/varnishd/cache/cache_vrt.c

```diff
@@ -719,12 +719,7 @@ VRT_Rollback(VRT_CTX, VCL_HTTP hp)
 		Req_Rollback(ctx->req);
 	} else if (hp == ctx->http_bereq) {
 		CHECK_OBJ_NOTNULL(ctx->bo, BUSYOBJ_MAGIC);
-		// -> VBO_Rollback ?
-		VCL_TaskLeave(ctx->bo->vcl, ctx->bo->privs);
-		VCL_TaskEnter(ctx->bo->vcl, ctx->bo->privs);
-		HTTP_Clone(ctx->bo->bereq, ctx->bo->bereq0);
-		WS_Reset(ctx->bo->bereq->ws, ctx->bo->ws_bo);
-		WS_Reset(ctx->bo->ws, ctx->bo->ws_bo);
+		Bereq_Rollback(ctx->bo);
```

Comment:
Rename to VBO_Rollback?

Reply:
yeah, names...
When I added the VBO_Rollback comment back in c98edbb0214f6d64b7822def58036cd67e3bdb1d, I would have thought the natural place for the code was `cache_busyobj.c`.
Now that it ended up in for proximity to the reused cleanup function, the natural way would be `VBF_Rollback`, I still thought `Bereq_Rollback` was clearer in analogy to `Bereq_Rollback`.

your choice?

Reply:
(damn sorry for all these edits, it's been a long day already)

Reply:
Calling it `VBO_Something` outside of `cache_busyobj.c` wouldn't be unprecedented, see `VCL_TaskEnter` for example.

Reply:
The VBO prefix is for code which works on the internals of busyobj, so to speak things that work on both req.body and beresp.body.